ConsumerApi.Tests.Integration/API/ChallengesApi.cs

```diff
@@ -10,9 +10,4 @@ internal async Task<HttpResponse<Challenge>> CreateChallenge(RequestConfiguratio
     {
         return await Post<Challenge>("/Challenges", requestConfiguration);
     }
-
-    internal async Task<HttpResponse<Challenge>> GetChallengeById(RequestConfiguration requestConfiguration, string id)
-    {
-        return await Get<Challenge>($"/Challenges/{id}", requestConfiguration);
-    }
```

Comment:
Why did you remove this?

Reply:
It is no longer used. I figured we'd want to only keep SDK methods in the future.

Reply:
Okay, you're right. 👍